firmware/README.asciidoc

```diff
@@ -0,0 +1,26 @@
+= Kivipallur Jurto firmware
+
+== Data to mainbord
+
+In order to be able to communicate between the computer and the mainbord. The speed values must be in the correct byte form.
+
+Python speed send comand:
+
+---- 
+andmed = struct.pack('<hhhHBH', speed1, speed2, speed3, thrower_speed, disable_failsafe, 0xAAAA)
+----
+
+* h - speed1, speed2, speed3 are int16_t
+* H - thrower_speed, disable_failsafe are uint16_t
+* 0xAAAA don't change
+* Thrower speed can be from 20 - 6000
+
+== Kivipallur Jurto software does not support feedback
+
+* Feedback was not neede for Kivipallur Jurtos software
+* Kivipallur Jurtos firmware has feedback support
```

Comment:
Firmware is sending data back and it should be documented. It's not important if the robot is using it or not.